index.js

```diff
@@ -0,0 +1,23 @@
+const express = require("express")
+const teams = require("./teams.json")
+
+const app = express()
+
+app.get("/teams", (request, response) => {
+  response.send(teams)
+})
+
+app.get("/teams/:input", (request, response) => {
+  const teamByParams = teams.filter(team => {
+    const requestInfo = request.params.input
```

Comment:
I would have done this outside the filter so it only happens once instead of 32 times. A quick destructure would also read better.

```JavaScript
const { input } = request.params
```